libraries/ESP8266WiFi/src/WiFiClientSecure.cpp

```diff
@@ -471,24 +463,11 @@ WiFiClientSecure::WiFiClientSecure(ClientContext* client, bool usePMEM, const ui
     _ssl->ref();
 
     if (usePMEM) {
-        // When using PMEM based certs, allocate stack and copy from flash to DRAM, call SSL functions to avoid
-        // heap fragmentation that would happen w/malloc()
-        uint8_t *stackData = (uint8_t*)alloca(max(certLen, rsakeyLen));
-        if (rsakey && rsakeyLen) {
-              memcpy_P(stackData, rsakey, rsakeyLen);
-              _ssl->loadServerRSAKey(stackData, rsakeyLen);
-        }
-        if (cert && certLen) {
-            memcpy_P(stackData, cert, certLen);
-            _ssl->loadServerX509Cert(stackData, certLen);
-        }
+        if (rsakey && rsakeyLen) _ssl->loadObject_P(SSL_OBJ_RSA_KEY, rsakey, rsakeyLen);
```

Comment:
Please no 1-line if statements. Just put the _ssl line indented under the if.

Reply:
My bad!  I try and cuddle all if{}s but sometimes one just slips by on me.  I've seen too many spots where folks have a 1-line, uncuddle'd if and decide "I'll put in a debug statement here" and end up changing the logic.